.github/workflows/hive.yaml

```diff
@@ -0,0 +1,239 @@
+
+env:
+  # Proxy
+  GOPROXY: "${{ vars.GOPROXY }}"
+  # Hive action environment variables
+  S3_BUCKET: hive-results
+  S3_PATH: fusaka
+  S3_PUBLIC_URL: https://hive.ethpandaops.io/fusaka/
+  INSTALL_RCLONE_VERSION: v1.68.2
+  EEST_BUILD_ARG_FIXTURES: https://github.com/ethereum/execution-spec-tests/releases/download/v4.1.0/fixtures_eip7692.tar.gz # TOD: change to new release
+  EEST_BUILD_ARG_BRANCH: main  # TODO: change to new release
+  # Flags used for all simulators
+  GLOBAL_EXTRA_FLAGS: >-
+    --client.checktimelimit=180s
+    --sim.parallelism=6
+    --docker.buildoutput
+  # Flags used for the ethereum/eest/consume-engine simulator
+  EEST_ENGINE_FLAGS: >-
+    --sim.buildarg fixtures=${EEST_BUILD_ARG_FIXTURES}
+    --sim.buildarg branch=${EEST_BUILD_ARG_BRANCH}
+    --sim.loglevel=3
+  # Flags used for the ethereum/eest/consume-rlp simulator
+  EEST_RLP_FLAGS: >-
+    --sim.buildarg fixtures=${EEST_BUILD_ARG_FIXTURES}
+    --sim.buildarg branch=${EEST_BUILD_ARG_BRANCH}
+    --sim.loglevel=3
+  # Flags used for the ethereum/rpc-compat simulator
+  RPC_COMPAT_FLAGS: >-
+    --sim.loglevel=3
+
+jobs:
+  prepare:
+    runs-on: ubuntu-latest
+    outputs:
+      # Hive version
+      hive_repo: ${{ steps.parse.outputs.hive_repo }}
+      hive_tag: ${{ steps.parse.outputs.hive_tag }}
+      # Versions of the clients
+      geth_repo: ${{ steps.parse.outputs.geth_repo }}
+      geth_tag: ${{ steps.parse.outputs.geth_tag }}
+      geth_docker_image_tag: ${{ steps.parse.outputs.geth_docker_image_tag }}
+      besu_repo: ${{ steps.parse.outputs.besu_repo }}
+      besu_tag: ${{ steps.parse.outputs.besu_tag }}
+      besu_docker_image_tag: ${{ steps.parse.outputs.besu_docker_image_tag }}
+      reth_repo: ${{ steps.parse.outputs.reth_repo }}
+      reth_tag: ${{ steps.parse.outputs.reth_tag }}
+      reth_docker_image_tag: ${{ steps.parse.outputs.reth_docker_image_tag }}
+      nethermind_repo: ${{ steps.parse.outputs.nethermind_repo }}
+      nethermind_tag: ${{ steps.parse.outputs.nethermind_tag }}
+      nethermind_docker_image_tag: ${{ steps.parse.outputs.nethermind_docker_image_tag }}
+      erigon_image: ${{ steps.parse.outputs.erigon_image }}
+      erigon_tag: ${{ steps.parse.outputs.erigon_tag }}
+      erigon_docker_image_tag: ${{ steps.parse.outputs.erigon_docker_image_tag }}
+      ethereumjs_repo: ${{ steps.parse.outputs.ethereumjs_repo }}
+      ethereumjs_tag: ${{ steps.parse.outputs.ethereumjs_tag }}
+      ethereumjs_docker_image_tag: ${{ steps.parse.outputs.ethereumjs_docker_image_tag }}
+      # client_config contains the YAML client config for Hive
+      client_config: ${{ steps.client_config.outputs.config }}
+    steps:
+      - id: parse
+        run: |
+          # Default client versions
+          GETH_DEFAULT="ethereum/go-ethereum@master"
+          BESU_DEFAULT="hyperledger/besu@main"
+          RETH_DEFAULT="paradigmxyz/reth@main"
+          NETHERMIND_DEFAULT="NethermindEth/nethermind@release/1.31.5"
```

Comment:
I guess you still need to add the eof branches here. Unfortunately, these are defined in two places.